ext/dom/characterdata.c

```diff
@@ -361,4 +365,105 @@ PHP_METHOD(domcharacterdata, replaceData)
 }
 /* }}} end dom_characterdata_replace_data */
 
+PHP_METHOD(domcharacterdata, remove)
+{
+	zval *id;
+	xmlNodePtr children, child;
+	dom_object *intern;
+	int stricterror;
+
+	id = ZEND_THIS;
+	if (zend_parse_parameters_none() == FAILURE) {
+		return;
```

Comment:
It would be very cool if `RETURN_THROWS()` was used everywhere where an exception is thrown (instead of the `return;`s and `RETURN_NULL();`s).